### PR TITLE
Added a self-script that toggles "test mode"

### PR DESCRIPTION
**Pull request by AfzalivE**
A "test mode" toggle that changes animation, transition, and animator scales to 0.0 when it's not 0.0 and back to 1.0 if it's 0.0.

Can be really helpful when running UI tests locally and you quickly need to disable animations.

### CODE REVIEW

File: self_scripts/toggle_test_mode.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import sys
+import os
+import subprocess
+
+adb_path = os.getenv('adb_path')
+serial = os.getenv('serial')
+
+get_window_animation_scale_cmd = "adb shell settings get global window_animation_scale"
+get_transition_animation_scale_cmd = "adb shell settings get global transition_animation_scale"
+get_animator_duration_scale_cmd = "adb shell settings get global animator_duration_scale"
```
Comment (zjn0505) [inline self_scripts/toggle_test_mode.py:11]: Duplicated commands.

Comment (AfzalivE) [reply self_scripts/toggle_test_mode.py:11]: Saw this in https://github.com/zjn0505/adb-alfred/blob/master/self_scripts/toggle_airplane_mode.py so I wasn't sure if I should just follow the same pattern or remove them. 

I'll remove them then.

File: self_scripts/toggle_test_mode.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+get_animator_duration_scale_cmd = "{0} -s {1} shell settings get global animator_duration_scale".format(adb_path, serial)
+
+get_cmds = [get_window_animation_scale_cmd, get_transition_animation_scale_cmd, get_animator_duration_scale_cmd]
+
+for cmd in get_cmds:  
+  result = subprocess.check_output(cmd, 
+              stderr=subprocess.STDOUT,
+              shell=True).strip()
+  mode = "0.0"
+  state = "false"
```
Comment (zjn0505) [inline self_scripts/toggle_test_mode.py:20]: `state` is not used.

File: self_scripts/toggle_test_mode.py [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  if result == "0.0":
+      mode = "1.0"
+      state = "true"
+
+put_window_animation_scale_cmd = "adb shell settings put global window_animation_scale 0.0"
+put_transition_animation_scale_cmd = "adb shell settings put global transition_animation_scale 0.0"
+put_animator_duration_scale_cmd = "adb shell settings put global animator_duration_scale 0.0"
```
Comment (zjn0505) [inline self_scripts/toggle_test_mode.py]: Duplicated commands

**Comment by zjn0505** [comment]
Nice PR.

If you have the needs, the modifier keys can also be passed to self scripts.
https://github.com/zjn0505/adb-alfred/blob/master/scripts/show_device_options.py#L236-L246

Then we can have "cmd", "opt", "ctrl", "fn" and "shift" as quick access to set different scales.

**Comment by zjn0505** [comment]
The script file name would be better named to indicate its animation scale function.

**Pull request merged**